catalog/spatial/registry/registry-identification-plugin/pom.xml

```diff
@@ -64,6 +59,11 @@
             <groupId>ddf.platform.util</groupId>
             <artifactId>platform-util</artifactId>
         </dependency>
+        <dependency>
+            <groupId>ddf.security</groupId>
+            <artifactId>ddf-security-common</artifactId>
+            <version>${project.version}</version>
```

Comment:
You should be able to remove the version here. This will come via dependencyManagement in the catalog/pom.xml file.